dts/bindings/pinctrl/st,stm32-pinctrl.yaml

```diff
@@ -1,11 +1,26 @@
 # Copyright (c) 2020 Linaro Limited
 # SPDX-License-Identifier: Apache-2.0
 
-description: STM32 Pin controller Node
+description: |
+    STM32 Pin controller Node
+    Based on pincfg-node.yaml binding.
+
+    Note: `bias-disable` and `drive-push-pull` are default pin configurations.
+           They will be applied in case no `bias-foo` or `driver-bar` properties
```

Comment:
I'm curious if we can do `default: true` in here for these?

Reply:
> I'm curious if we can do `default: true` in here for these?

No; `default: true` can't be made to work. https://github.com/zephyrproject-rtos/zephyr/pull/31366#issuecomment-761130675